sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -673,7 +675,7 @@ This project has adopted the [Microsoft Open Source Code of Conduct][code_of_con
 [ta_pypi]: https://pypi.org/project/azure-ai-textanalytics/
 [ta_ref_docs]: https://aka.ms/azsdk-python-textanalytics-ref-docs
 [ta_samples]: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/textanalytics/azure-ai-textanalytics/samples
-[ta_product_documentation]: https://docs.microsoft.com/azure/cognitive-services/text-analytics/overview
+[language_product_documentation]: https://docs.microsoft.com/azure/cognitive-services/language-service/overview
```

Comment:
nit: I wonder if it is better to drop the `overview`. It has a good layout of the functionalities and if someone wants to drill more then they click in `Overview`

Reply:
sure, that sounds good